code/modules/food/recipes_microwave.dm

```diff
@@ -1747,30 +1747,74 @@
 		)
 	result = /obj/item/weapon/reagent_containers/food/snacks/flammkuchen
 
-// Currently Disabled //////////////////////////////////////////
```

Comment:
I considered that, and I've passed up this particular block of comments for a long time. But it has been many, many years and no one has even requested the return of banana phones. Someone can write the recipe from scratch later (not exactly a difficult proposition) if it ever returns, it doesn't even exist in the code elsewhere AFAIK.